auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -403,32 +398,61 @@ public Account getAccountDetails(String actorCrn, String accountId, Optional<Str
         }
     }
 
-    @Cacheable(cacheNames = "umsUserRoleAssigmentsCache", key = "{ #actorCrn, #userCrn }")
+    @Cacheable(cacheNames = "umsUserRoleAssigmentsCache", key = "{ #userCrn }")
     public List<UserManagementProto.ResourceAssignment> listResourceRoleAssigments(String actorCrn, String userCrn, Optional<String> requestId) {
         try (ManagedChannelWrapper channelWrapper = makeWrapper()) {
             UmsClient client = makeClient(channelWrapper.getChannel(), actorCrn);
             return client.listAssigmentsOfUser(requestId.orElse(UUID.randomUUID().toString()), userCrn);
         }
     }
 
-    //"#value.concat(#fieldId).concat(#projectId)"
-//    @Cacheable(cacheNames = "umsUserRightsCache", key = "{ #actorCrn, #userCrn, #right, #resource }")
-    @Cacheable(cacheNames = "umsUserRightsCache", key = "{ #userCrn, #right, #resource }")
-    public boolean checkRight(String actorCrn, String userCrn, String right, String resource, Optional<String> requestId) {
+    @Cacheable(cacheNames = "umsUserHasRightsForResourceCache", key = "{ #userCrn, #right, #resource }")
```

Comment:
Is it intentional that you want to use same cache as used for `hasRights `?

Reply:
is there a meaningful difference on the ask towards ums from a caching point of view?

Reply:
i guess not, only the name of the cache deceived me, but it's not important